run.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+if [ ! -v OPTIMADE_CONFIG_FILE ] || [ -z "$OPTIMADE_CONFIG_FILE" ]; then
```

Comment:
```suggestion
if [ -z "$OPTIMADE_CONFIG_FILE" ]; then
```
The "-v" flag is a quite recent addition in bash (>=4.2), and even quite recent Mac OSX computers are still on bash 3. The standard way of doing this in the past has been to just check if the variable evaluates to empty, which works as long as we are not in `set -u` mode.

Reply:
@ml-evs Are you ok with this change? We had a user trying out run.sh and getting "./run.sh: line 3: [: -v: unary operator expected"...

Reply:
Thanks for pointing this out, had no idea they were that far behind!